addons/website/models/res_config.py

```diff
@@ -54,6 +55,25 @@ def _default_website(self):
     has_google_analytics = fields.Boolean("Google Analytics")
     has_google_analytics_dashboard = fields.Boolean("Google Analytics in Dashboard")
     has_google_maps = fields.Boolean("Google Maps")
+    auth_signup_uninvited = fields.Selection([
+        ('b2b', 'On invitation (B2B)'),
+        ('b2c', 'Free sign up (B2C)'),
+    ], string='Customer Account')
+
+    @api.model
+    def get_default_auth_signup_uninvited(self, fields):
+        # we use safe_eval on the result, since the value of the parameter is a nonempty string
+        allow_uninvited = safe_eval(
```

Comment:
Please fix this. As rule, you must not use safe_eval/eval unless its power is truly needed, i.e. when you need to evaluate arbitrary expressions within a sandbox. And then, with great care.
Whether your code is safe does not matter, it's a rule :-)
Not to mention that other parts of the code [already use literal_eval](https://github.com/odoo-dev/odoo/blob/349de6ddacc23f09dda557399881e9979374f83e/addons/auth_signup/models/res_users.py#L87) (and even that is questionable, because frankly, `value.lower() != "true"` is quite enough for our purpose).